packages/ifpack2/src/Ifpack2_Details_FastILU_Base_def.hpp

```diff
@@ -93,6 +95,12 @@ apply (const Tpetra::MultiVector<Scalar,LocalOrdinal,GlobalOrdinal,Node> &X,
        Scalar alpha,
        Scalar beta) const
 {
+  const std::string timerName ("Ifpack2::FastILU::apply");
+  Teuchos::RCP<Teuchos::Time> timer = Teuchos::TimeMonitor::lookupCounter (timerName);
+  if (timer.is_null ()) {
+    timer = Teuchos::TimeMonitor::getNewCounter (timerName);
```

Comment:
getNewCounter should just look up the existing timer if it exists, so you could just call getNewCounter.

Reply:
Not wrong, but you could write this in one line with getNewCounter.